terracumber_config/tf_files/Uyuni-Master-refenv-NUE.tf

```diff
@@ -0,0 +1,194 @@
+  type = "string"
+}
+
+provider "libvirt" {
+  uri = "qemu+tcp://cokerunner.mgr.suse.de/system"
+}
+
+module "base" {
+  source = "./modules/base"
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+
+  name_prefix = "uyuni-refhead-"
+  use_avahi   = false
+  domain      = "mgr.suse.de"
+  images      = ["centos7o", "sles15sp1o", "sles15sp2o", "sles15sp3o", "ubuntu2004o"]
+  provider_settings = {
+    pool         = "ssd"
+    network_name = null
+    bridge       = "br2"
+  }
+}
+
+module "server" {
+  source             = "./modules/server"
+  base_configuration = module.base.configuration
+  product_version    = "head"
+  name               = "srv"
+
+  use_os_released_updates = true
+  disable_download_tokens = false
+  from_email              = "root@suse.de"
+  postgres_log_min_duration = 0
+  channels                = ["sle-product-sles15-sp3-pool-x86_64", "sle-product-sles15-sp3-updates-x86_64", "sle-module-basesystem15-sp3-pool-x86_64", "sle-module-basesystem15-sp3-updates-x86_64", "sle-module-containers15-sp3-pool-x86_64", "sle-module-containers15-sp3-updates-x86_64", "sle-manager-tools15-pool-x86_64-sp3", "sle-manager-tools15-updates-x86_64-sp3", "sle-module-server-applications15-sp1-pool-x86_64", "sle-module-server-applications15-sp1-updates-x86_64"]
+
+  provider_settings = {
+    mac = "aa:b2:93:01:00:e1"
+    memory = 8192
+  }
+}
+
+module "suse-client" {
+  source             = "./modules/client"
+  base_configuration = module.base.configuration
+  product_version    = "head"
+  name               = "cli-sles15"
+  image              = "sles15sp3o"
+
+  server_configuration    = module.server.configuration
+  use_os_released_updates = true
+
+  provider_settings = {
+    mac = "aa:b2:93:01:00:e4"
+  }
+}
+
+module "suse-minion" {
+  source             = "./modules/minion"
+  base_configuration = module.base.configuration
+  product_version    = "head"
+  name               = "min-sles15"
+  image              = "sles15sp3o"
+
+  server_configuration    = module.server.configuration
+  use_os_released_updates = true
+
+  provider_settings = {
+    mac = "aa:b2:93:01:00:e6"
+  }
+}
+
+module "redhat-minion" {
+  source               = "./modules/minion"
+  base_configuration   = module.base.configuration
+  product_version      = "head"
+  name                 = "min-centos7"
+  image                = "centos7o"
+  server_configuration = module.server.configuration
+
+  provider_settings = {
+    mac = "aa:b2:93:01:00:e9"
+    // Since start of May we have problems with the instance not booting after a restart if there is only a CPU and only 1024Mb for RAM
+    // Also, openscap cannot run with less than 1.25 GB of RAM
+    memory = 2048
+    vcpu = 2
+  }
+}
+
+module "debian-minion" {
+  source               = "./modules/minion"
+  base_configuration   = module.base.configuration
+  product_version      = "head"
+  name                 = "min-ubuntu1804"
+  image                = "ubuntu2004o"
```

Comment:
`ubuntu1804o`

Reply:
Just copied this over from
https://github.com/SUSE/susemanager-ci/blob/271f4d2f1a2ccc3853ef8ce31777da7389b9545c/terracumber_config/tf_files/SUSEManager-Head-refenv-NUE.tf#L157-L163
Maybe this should be adjusted there, too?

Reply:
That looks wrong to me, indeed.
Maybe they did it in that way, to don't spend another domain? No idea.
@juliogonzalez did you noticed this? ^^^